### PR TITLE
Add a kubernetes file for the web demo.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
@joelgrus any suggestions on where to organize the yaml files?  They are starting to clutter the root.

**Comment by schmmd** [comment]
With this yaml, you can start a production demo with `kubectl create -f kubernetes-webdemo.yaml`.

**Comment by joelgrus** [comment]
there's only two so far? that's fine. if we start having more, maybe make a `kube` directory

**Comment by schmmd** [comment]
@matt-gardner @DeNeutoy why is this failing the code coverage plugin?  Do we have these settings turned up too high?

**Comment by matt-gardner** [comment]
It's failing the % diff covered target, because of the two python lines that you changes, 0 were covered by tests.  I think 90% coverage is a good target to strive for in python code, but falling below that doesn't mean you can't merge the PR - it's not a `Required` check.

**Comment by schmmd** [comment]
ok, I will add another PR to the wall of shame: https://github.com/allenai/allennlp/pulls

File: Dockerfile.cpu [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:16.04
 
+EXPOSE 8000
```
Comment (DeNeutoy) [inline Dockerfile.cpu]: Will these eventually change to `PUBLISH` is the idea that people submit requests via the website which we process privately?

Comment (schmmd) [inline Dockerfile.cpu]: hmmm... I don't quite understand.  Let's follow up in person tomorrow!

**Pull request merged**